app/theme/client/imports/components/table.css

```diff
@@ -215,6 +215,8 @@
 	font-size: 0.75rem;
 	font-weight: 500;
 	line-height: 1rem;
+	background-color: white;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected an empty line before property "background-color" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5001518)

Reply:
yeah, I fixed this. hope it works fine now